src/main/java/org/folio/service/mapping/MappingServiceImpl.java

```diff
@@ -29,6 +34,8 @@
 @Service
 public class MappingServiceImpl implements MappingService {
   private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  private static final int MAPPING_POOL_SIZE = 4;
```

Comment:
Is there a reason you choose 4? If I understand NewWork StealingPool by default uses the number of available processors to the JVM

Reply:
Also given that this module can be run in multiple environments like container bare metals, selecting such number would have different effects based on where it is deployed

Reply:
The reason to use exactly 4 threads is that it is enough 4 threads to noticeably increase the productivity of the MappingService and not enough to load-up all the available processors. 4 threads is a trade-off that leaves CPU resources to handle other operations. The more threads in a pool the more consuming time to manage these threads. It tested Mapping Service using 16 threads and it works worse than using 4.